src/pages/docs/spoke/for-spoke-admins/billing-&-pricing.astro

```diff
@@ -0,0 +1,74 @@
+---
+import Layout from "../../../../layouts/Layout.astro";
+import "animate.css";
+---
+
+<Layout title="With the Ranks">
+	<main
+		class="animate__animated animate__fadeIn animate__slow py-20 md:py-40"
+	>
+		<div class="flex items-center flex-col px-20">
+			<h2 class="text-center">Billing & Pricing</h2>
+			<h3>Billing Periods</h3>
+			<p>
+				Usage-based products (e.g., SMS segments, MMS segments) are
+				billed at the end of each month. License-based products (e.g.,
+				consulting, database services) are billed at the top of the
+				month.
+			</p>
+			<h3>Payment Methods</h3>
+			<p>
+				Invoices are sent via email and payment by ACH is preferred. If
+				you are unable to pay by ACH, please <a
+					href="mailto:support@withtheranks.com">email us</a
+				> and mention your preferred payment method. We will accept credit
+				cards upon request, but a 3% processing fee may apply.
+			</p>
+			<h3>Usage Reports</h3>
+			<p>
+				If you require an ad-hoc usage report mid-month, please <a
+					href="mailto:support@withtheranks.com">email us</a
+				> and include the name of your organization and the time period desired.
+			</p>
+			<h3>Pricing</h3>
+			<h5>Texts</h5>
+			<p>
+				Under our Standard pricing model, outbound SMS segments are
+				priced at 1.25¢ and <a
+					href="https://withtheranks.com/for-spoke-admins/include-an-image-in-a-message"
+					>MMS</a
+				> segments are priced at 3¢. You can read more about segments <a
+					href="https://withtheranks.com/for-spoke-admins/segments-and-encodings"
+					>here</a
+				>.
+			</p>
+			<h5>Number Lookups</h5>
+			<p>
+				Contact validation is priced at 1/4¢ ($.0025) per cell. Use our
+				<!-- TODO: update href with a link to the specific section if possible -->
+				<a
+					href="https://withtheranks.com/for-spoke-admins/build-a-campaign"
+					>Filter Landlines</a
+				> feature to confirm that your contact lists are textable!
+			</p>
+			<h5>Consulting Services</h5>
+			<p>
+				To inquire about our consulting offerings, please <a
+					href="mailto:support@withtheranks.com">email us</a
+				> or book a <a href="https://calendly.com/with-the-ranks"
+					>time to chat</a
+				>! Our team, former leaders of the Bernie 2020 National Texting
+				Team and Tech Team, can offer extra assistance in planning,
+				building, and analyzing your campaigns.
```

Comment:
Flagging copy on our connection to Bernie here. The previous copy that Rewired used was

> Our team of organizers, former leaders of the Bernie 2020
> 				National Texting Team, offer extra assistance in planning,
> 				building, and analyzing your campaigns